schema.json

```diff
@@ -422,27 +422,31 @@
           "const": "repeat"
         },
         "value": {
-          "description": "A repeat argument value, there can be a nested repeat arguments inside.",
+          "description": "A repeat argument value.",
           "type": "array",
           "items": {
             "oneOf": [
```

Comment:
Do we need this `oneOf`?

Reply:
Yeah that's the outer array for the repeat.

Reply:
Can it just be:
```json
{
  "description": "A repeat argument value.",
  "type": "array",
  "items": {
    "type": "array",
    "items": {
      "oneOf": [
        {
          "$ref": "#/$defs/string_argument"
        },
        {
          "$ref": "#/$defs/number_argument"
        },
        {
          "$ref": "#/$defs/boolean_argument"
        },
        {
          "$ref": "#/$defs/symbol_argument"
        },
        {
          "$ref": "#/$defs/hex_argument"
        }
      ]
    }
  }
}
```
Just wondering if the outer array needs the `oneOf` or if it works without?

Reply:
It looks like that's invalid because `items` needs to be an object not an array.